src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/MemoryTests.cs

```diff
@@ -24,14 +24,17 @@ public MemoryTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buil
     [InlineData("Release", false)]
     public async Task AllocateLargeHeapThenRepeatedlyInterop(string config, bool buildNative)
     {
-        // native build triggers passing value form EmccMaximumHeapSize to MAXIMUM_MEMORY that is set in emscripten
-        // in non-native build EmccMaximumHeapSize does not have an effect, so the test will fail with "out of memory"
         CopyTestAsset("WasmBasicTestApp", "MemoryTests", "App");
-        string extraArgs = $"-p:EmccMaximumHeapSize=4294901760 -p:WasmBuildNative={buildNative}";
+        string extraArgs = $"-p:EmccMaximumHeapSize=4294901760";
+        if (buildNative)
```

Comment:
Do we still need the test variant with `WasmBuildNative=true`?

Reply:
Actually, not. It was never the problematic case.

Reply:
I cannot see `MemoryTests` or generally any WBT running in `no-workload` mode. There's only `Workloads-NoWebcil-ST-Wasm.Build` and `Workloads-ST-Wasm.Build.Tests`. Do you know why?

Reply:
With no-workload, all tests run as a single helix job and category "no-workload" filter is applied

Reply:
https://github.com/dotnet/runtime/blob/main/src/libraries/sendtohelix-browser.targets#L141

Reply:
Thanks. It seems the OOM happens when we have no workload, I added the test with the proper filter.

Reply:
when we have no workload, we can't native relink and the `EmccMaximumHeapSize` does nothing and that's why it OOM ?

Reply:
Because we do not relink and effectively, we don't set the EmccFlag. That's why. I have to track the flow to check what failed and why we did not get no-workload-run-restore error.